doc/man3/X509_VERIFY_PARAM_set_flags.pod

```diff
@@ -311,6 +315,8 @@ requirements and lead to a locally trusted root.
 This is especially important when some certificates in the trust store have
 explicit trust settings (see "TRUST SETTINGS" in L<openssl-x509(1)>).
 
+=begin comment
+
```

Comment:
Do not just comment out but remove. However this sounds like something for a separate PR targeting just master branch.